tests/desktop/consumer_pages/test_details_page.py

```diff
@@ -14,6 +14,9 @@ class TestDetailsPage(BaseTest):
     @pytest.mark.sanity
     @pytest.mark.nondestructive
     def test_that_application_page_contains_proper_objects(self, mozwebqa):
+        if '-dev' in mozwebqa.base_url:
+            pytest.xfail("Bug 1242648 - Support info doesn't show up on an empty profile")
```

Comment:
Thanks @davehunt. After some experimentation it looks like:

```python
    @pytest.mark.xfail("'-dev' in pytest.config.option.base_url",
                       reason="Bug 1242648 - Support info doesn't show up on an empty profile")
```

will work, but then, when testing it out I found the test has started passing, so I'm going to close this PR as we no longer seem to need it. It looks like maybe they removed some invalid apps from dev.